src/shared/utils/pagination.util.ts

```diff
@@ -0,0 +1,19 @@
+export function getPaginationOffset(page: number | null, limit: number | null) {
+  return {
+    take: limit ?? 20,
+    skip: page && page > 0 ? (page - 1) * limit : 0,
+  };
+}
+
+export function getPaginationMetadata(
+  page: number | null,
+  limit: number | null,
+  total: number | null,
```

Comment:
Why can these be null?

Reply:
because they are not always provided, I mean optional